ros_buildfarm/templates/snippet/plot_plugin.xml.em

```diff
@@ -3,6 +3,7 @@
 @[for plot_group, plot_list in plots.items()]@
 @[for plot in plot_list]@
         <hudson.plugins.plot.Plot>
+          <description>@(plot.description.replace('<', '&lt;').replace('<', '&gt;'))</description>
```

Comment:
I don't think this is sufficient. How about `&`, single/double quotes, etc.?

Reply:
How about using an existing function to perform the escaping instead of doing it manually?

Reply:
I was trying to use :

```
import html; html.escape(plot.description)
```

But I got some failures. I decided to do it manually.

Reply:
> But I got some failures.

Can you be more specific? Was it a template issue?

I've seen problems importing in templates before, and have found ways to [work around them](https://github.com/ros-infrastructure/ros_buildfarm/blob/117f59ac7a7b7c69e943300204c2789e5fc77039/ros_buildfarm/templates/ci/create_workspace.Dockerfile.em#L2-L5).

Reply:
Thanks for the tip!